test/server.test.js

```diff
@@ -112,3 +112,46 @@ test('automatic default route', t => {
     })
   })
 })
+
+test('maps two routes when trailing slash should be trimmed', t => {
+  t.plan(13)
+  const findMyWay = FindMyWay({
+    trimTrailingSlash: true
+  })
+
+  findMyWay.on('GET', '/test/', (req, res, params) => {
```

Comment:
Can you also add another route where you register `/othertest` (without the trailing slash) and test how it behaves?
Then it could be useful add a different test with `trimTrailingSlash` set to `false` and test how it behaves as well. :)